.config-schema.json

```
@@ -430,6 +430,29 @@
                     "x-env-variable": "OPENFGA_CHECK_QUERY_CACHE_TTL"
                 }
             }
+        },
+        "dispatchThrottling": {
+            "type": "object",
+            "properties": {
+                "enabled": {
+                    "description": "enable throttling when request's number of dispatches is high",
+                    "type": "bool",
+                    "default": "false",
+                    "x-env-variable": "OPENFGA_DISPATCH_THROTTLING_ENABLED"
+                },
+                "timeTickerFrequency": {
+                    "description": "defines how frequent dispatch throttling will be evaluated",
+                    "type": "duration",
+                    "default": "10µs",
+                    "x-env-variable": "OPENFGA_DISPATCH_THROTTLING_TIME_TICKER_FREQUENCY"
+                },
+                "level": {
+                    "description": "define the number of dispatches to be throttled",
```

"Dispatch" is not a term that we use externally. I'm not sure if it makes sense to introduce it now or use a term that conveys a similar meaning. 

Perhaps something like:

```suggestion
                    "description": "define the number of recursive operations to occur before getting throttled",
```

---

updated